src/nvim/CMakeLists.txt

```diff
@@ -118,6 +118,10 @@ endforeach()
 
 list(REMOVE_ITEM NVIM_SOURCES ${to_remove})
 
+if (NOT WIN32)
+  list(REMOVE_ITEM NVIM_HEADERS "${PROJECT_SOURCE_DIR}/src/nvim/os/pty_process_win.h")
+endif()
```

Comment:
@erw7 Do you happen to remember why this was needed? I removed it in https://github.com/neovim/neovim/pull/7007/commits/b885b9358a4f1b0eb38a75b4a31839ba23346be3 and it seems to build OK on linux, bsd, and macOS.